src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -3,57 +3,45 @@ package nl.knaw.dans.easy.stage
 import java.io.File
 
 import org.joda.time.DateTime
-import org.rogach.scallop.ScallopConf
+import org.rogach.scallop.{singleArgConverter, ValueConverter, ScallopConf}
 
-class Conf(args: Seq[String]) extends ScallopConf(args) {
-  printedName = "easy-state-dataset"
+class Conf(args: Seq[String] = "-b. -s. -t2015 -uu -dd".split(" ")) extends ScallopConf(args) {
```

Comment:
Bit of a convoluted and implicit way to set defaults for some of the parameters. Also, I am not sure what -uu and -dd actually will do.